solar_car_process.py

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env python3
 
+import sys
 import subprocess
+import logging
+import shlex
+import time
```

Comment:
Generally, imports that are grouped together like these ones should be in alphabetical order.